site/content/en/docs/manual/advanced/annotation-with-brushing-tools.md

```diff
@@ -0,0 +1,121 @@
+---
+title: 'Annotation with brushing tools'
+linkTitle: 'Annotation with brushing tools'
+weight: 13
+description: 'Guide to annotating tasks using  brushing tools.'
+---
+
+The brushing tool creates a segmentation mask (further - mask) on the top of the image. Use it, when you need to add a label to an object that has several separated parts, but must be counted as one, for example - a house hiding behind trees or a pillar behind a traffic sign.
```

Comment:
actually, not on the top of the image

Reply:
I suggest to put it this way: 

```suggestion
The brush tool allows you to draw mask annotations. This type of annotations is mainly used in Semantic (Panoptic) and Instance Image Segmentation tasks. You can read more about working with segmentation masks in CVAT [here](/docs/manual/advanced/annotation-with-polygons/creating-mask/).

Comparing to polygons, this tool can be particularly useful to create masks for disjoint objects, having multiple parts, such as a house hiding behind trees, a car occluded by a pedestrian, or a pillar behind a traffic sign. In some cases, it can also be simpler and faster to draw a mask with a brush than a polygon. The brush tool has different modes that can be useful in different annotation scenarios. For instance, there are modes to erase pixels, change brush shapes and to draw polygon-like contours.
```

Reply:
By a number of style manuals, It is advised to avoid "allows"(advice from multiple manuals of styles), And  "comparing to polygons" sounds a bit confusing. The readers might know nothing about polygon tools and we do not want them to start thinking that he/she has to learn what is polygon first. Or do we?

Reply:
We also must avoid passive voice as much as possible (it is just a piece of general writing advice).